### PR TITLE
Minor description edits for clarity

### PR DESCRIPTION
**Pull request by todb-r7**
Edited modules/exploits/multi/browser/adobe_flash_nellymoser_bof.rb first landed in #5642, Adobe Flash CVE-2015-3113 Nellymoser Audio Decoding BOF

Edited modules/post/windows/gather/credentials/enum_laps.rb first landed in #5590, @Meatballs1 adds MS LAPS Enum post mod

Edited modules/post/windows/gather/enum_ad_bitlocker.rb first landed in #5268, @Meatballs1's post windows module to retrieve Bitlocker Recovery Keys from AD

### CODE REVIEW

**Comment by todb-r7** [comment]
Dang my commit message is wonky.

**Comment by wvu-r7** [comment]
Rebasing heathen!

**Comment by todb-r7** [comment]
haha did it again.

Turns out my PR number was getting line wrapped so the line started with a #.

oops.

File: modules/post/windows/gather/credentials/enum_laps.rb [hunk 1]
```diff
@@ -22,9 +22,9 @@ def initialize(info={})
       'Name'         => 'Windows Gather Credentials Local Administrator Password Solution',
       'Description'  => %Q{
         This module will recover the LAPS (Local Administrator Password Solution) passwords,
-        configured in active directory. Note, only privileged users should be able to access
-        these fields. Note: The local administrator account name is not stored in active directory,
-        so we assume that this will be 'Administrator' by default.
+        configured in Active Directory, which is usually only accessable by privileged users.
```
Comment (wvu-r7) [inline modules/post/windows/gather/credentials/enum_laps.rb]: s/accessable/accessible/ # :)

File: modules/exploits/multi/browser/adobe_flash_nellymoser_bof.rb [hunk 2]
```diff
@@ -20,9 +20,14 @@ def initialize(info={})
 
         Windows 7 SP1 (32-bit), IE11 and Adobe Flash 18.0.0.160,
         Windows 7 SP1 (32-bit), Firefox 38.0.5 and Adobe Flash 18.0.0.160,
-        Windows 8.1, Firefox 38.0.5 and Adobe Flash 18.0.0.160, and
-        Linux Mint "Rebecca" (32 bits), Firefox 33.0 and Adobe Flash 11.2.202.466.
+        Windows 8.1, Firefox 38.0.5 and Adobe Flash 18.0.0.160,
+        Linux Mint "Rebecca" (32 bits), Firefox 33.0 and Adobe Flash 11.2.202.466, and
         Ubuntu 14.04.2 LTS, Firefox 35.01, and Adobe Flash 11.2.202.466.
+
+
```
Comment (wvu-r7) [inline modules/exploits/multi/browser/adobe_flash_nellymoser_bof.rb]: Double newline.

**Comment by wvu-r7** [comment]
Merged.

**Comment by wvu-r7** [comment]
https://github.com/rapid7/metasploit-framework/commit/8892cbdd102870a9857687662a6f794b6fdf994a

**Comment by wvu-r7** [comment]
Looks like I missed the second rebased commit. Doh!

**Pull request closed without merging**